ArmRobot/Assets/Scripts/RobotAgent.cs

```diff
@@ -40,46 +40,66 @@ public override void CollectObservations(VectorSensor sensor)
         float[] rotations = robotController.GetCurrentJointRotations();
         foreach (float rotation in rotations)
         {
-            // normalize rotation to [0, 1] range
-            float normalizedRotation = rotation / 360.0f;
+            // normalize rotation to [-1, 1] range
+            float normalizedRotation = (rotation / 360.0f) %  1f;
```

Comment:
Why did we add the modulo 1?

Reply:
Because a rotation of x + 360 = x (corresponds to a full rotation)
It has the advantage to compress the state space better at the cost of having the observation space be discontinuous at 359 degrees

Reply:
It is also discontinuous at 0, right in the middle of our [-1, 1] range, right? Using python as a sanity check, I got:
```
>>> 0.001 % 1
0.001
>>> -0.001 % 1
0.999
```
Also, I think we might be losing some information in our [-1, 1] range:
```
>>> -0.5 % 1
0.5
>>> 0.5 % 1
0.5
```

Reply:
What if we make the rotation range be [0, 1], with the discontinuity at 0/1, but then shift the range to map the current 0 position to 0.5, so that at least there is not a discontinuity at the default position?

Reply:
Or - if we want to be more realistic - we could physically limit the rotation of each joint to [-720, 720 ] degrees (this is the limit on the real robot). Commands to continue rotating past the limit simply don't work. Then we have an enforced range we can normalize over without worrying about discontinuities.

Reply:
C# and Python behave differently. -0.5f % 1f = -0.5f not 0.5f (I don't know why)

Reply:
Oh wow, I didn't know that. Ok then. :)